src/main/java/org/broadinstitute/hellbender/tools/walkers/sv/SVConcordance.java

```diff
@@ -50,6 +52,8 @@
  * of the specific fields. For multi-allelic CNVs, only a copy state concordance metric is
  * annotated. Allele frequencies will be recalculated automatically if unavailable in the provided VCFs.
  *
+ * Note that the output is unsorted.
```

Comment:
This is only if the `do-not-sort` param is set, right?

Reply:
Thanks for catching this. Originally I had the default behavior switched and forgot to change this.